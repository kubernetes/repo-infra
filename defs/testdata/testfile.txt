This is just some data to hash
