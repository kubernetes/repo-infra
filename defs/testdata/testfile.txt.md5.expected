6c840392943ddfc0d213a8786ddcf913
